justin-tan/week2/fri/ga-bank/js/atm.js

```diff
@@ -0,0 +1,220 @@
+
+
+//event handlers for checking account background color
+$('#checking-withdraw').on( 'click', backgroundCheckingColor );
+$('#checking-deposit').on( 'click', backgroundCheckingColor );
+
+
+//=========================================================
+//=========================================================
+//=========================================================
+
+
+//function for SAVINGS deposit
+const savingsDeposit = function() {
+    //set input value into a number
+    const $savingsValue = $('#savings-amount').val();
+    const $savingsAmount = parseInt($savingsValue);
+
+    //add input amount into account balance
+    $savingsBalance = $savingsBalance + $savingsAmount;
+    $('#savings-balance').text(`$${$savingsBalance}`);
+}
+
+//===========================================================
+
+
+//function for SAVINGS withdrawal
+const savingsWithdrawal = function() {
+    //set variable for total balance
+    let totalBalance = $checkingBalance + $savingsBalance
+
+    //set input value into a number
+    const $savingsValue = $('#savings-amount').val();
+    const $savingsAmount = parseInt($savingsValue);
+    
+    //set conditions withdrawal:
+    //when you have sufficient balance to initiate withdraw
+    if ( $savingsBalance >= $savingsAmount ) {
+        $savingsBalance = $savingsBalance - $savingsAmount;
+        $('#savings-balance').text(`$${$savingsBalance}`);
+    
+    //when overdraft protection initiated
+    } else if ( $savingsBalance < $savingsAmount && $savingsAmount <= totalBalance) {
+        
+        //set variable for overdraft
+        let overdraft = $savingsBalance - $savingsAmount;
+
+        //formula for remaining balances for both account
+        $savingsBalance = $savingsBalance - $savingsBalance;
+        $checkingBalance = $checkingBalance + overdraft;
+        $('#checking-balance').text(`$${$checkingBalance}`) ;
+        $('#savings-balance').text(`$${$savingsBalance}`);
+
+    //when you have insufficient total balance to withdraw
+    } else if ( $savingsAmount > totalBalance ) {
+        $('#savings-balance').text(`Insufficient $$`);
+    }
+}
+
+//===========================================================
+
+//function for SAVINGS BALANCE background color
+const backgroundSavingsColor = function () {
+    if ( ($savingsBalance === 0 || $(`#savings-balance`).text() === `Insufficient $$`) ) {
+        $('#savings-balance').css('background-color', 'red');
+    } else {
+        $('#savings-balance').css('background-color', 'rgb(227, 227, 227');
+    }
+}
+
+//============================================================
+
+//event handlers for savings account deposit and withdrawal buttons
+$('#savings-deposit').on( 'click', savingsDeposit );
+$('#savings-withdraw').on( 'click', savingsWithdrawal );
+
+//event handlers for checking account background color
+$('#savings-withdraw').on( 'click', backgroundSavingsColor );
+$('#savings-deposit').on( 'click', backgroundSavingsColor );
```

Comment:
you could potentially put the background colour function inside the withdraw function. its just another if statement or additional statement inside an existing if statement, then you add the css line within that